code/src/java/pcgen/output/channel/ChannelUtilities.java

```diff
@@ -35,14 +37,44 @@
  */
 public class ChannelUtilities
 {
-	private static final VariableLibraryFacet VARLIB_FACET = FacetLibrary
-		.getFacet(VariableLibraryFacet.class);
-	private static final ScopeFacet SCOPE_FACET = FacetLibrary
-		.getFacet(ScopeFacet.class);
-	private static final VariableStoreFacet RESULT_FACET = FacetLibrary
-		.getFacet(VariableStoreFacet.class);
-	private static final SolverManagerFacet MGR_FACET = FacetLibrary
-		.getFacet(SolverManagerFacet.class);
+	private static final VariableLibraryFacet VARLIB_FACET =
```

Comment:
if I wasn't guilty of the same things I'd get annoyed at formatting and code changes in the same commit :-p